docs/Tutorials/CrossCompilationSetup/macOS.md

```diff
@@ -1,6 +1,6 @@
-# Mac OS X
+# macOS
 
-MacOS like Linux is a unix system and thus may be used directly for most of this 
+macOS, like Linux, is a Unix system and thus may be used directly for most of this 
```

Comment:
this isn't technically correct. macOS _is_ a Unix. Linux is not. But fixing that isn't a spelling thing...